MD/md-71/README.md

```diff
@@ -0,0 +1,39 @@
+# MD-71: Informer service for the Lock/Mint-based Native Bridge
+- **Description**: Provide a component that informs about the total circulating supply of \$MOVE across L1 and L2.
+- **Authors**: [Author](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Overview
+
+Several components should react if the bridge is under attack or faulty. In particular, the considered components are the Security Fund, see [MIP-50](https://github.com/movementlabsxyz/MIP/pull/50) and the Rate Limiter, see [MIP-56](https://github.com/movementlabsxyz/MIP/pull/56), and more generally the governance operator.
+
+The Operator that controls these components requires knowledge about the states of minted or locked tokens on L1 and L2. Moreover, the operation of these components may be handled via a governance, which could also rely on state information.
```

Comment:
Make clear that the Operator is a human with abilities to sign transactions, to send MOVE from the Insurance Fund.

Reply:
added.